app/controllers/sites/new.js

```diff
@@ -27,17 +27,16 @@ export default class CreateSitesNewController extends Controller {
   @tracked isPrimarySite = false;
   @tracked validationErrors = {};
   @tracked validationWarnings = {};
-  @tracked showWarningModal = false;
+  @tracked buttonCounter = 0;
```

Comment:
Zou deze variable misschien beter iets als saveButtonPressed noemen. Is duidelijker dan over welke button het gaar.